src/site/stages/build/drupal/graphql/vetCenter.graphql.js

```diff
@@ -0,0 +1,83 @@
+const vetCenterFragment = `
+ fragment vetCenterFragment on NodeVetCenter {
+        entityId
+        entityUrl {
+          path
+          routed
+        }
+        entityBundle
+        entityLabel
+        fieldIntroText
+        fieldOfficeHours {
+          day
+          starthours
+          endhours
+          comment
+        }
+        # Other locations link:
+        reverseFieldOfficeNode(limit: 500, filter:{conditions: [{field: "type", value: ["vet_center_locations_list"]}]}) {
+          entities {
+            ... on NodeVetCenterLocationsList {
+              entityUrl {
+                path
+                routed
+              }
+            }
+          }
+        }        
+        fieldPrepareForVisit {
+          entity {
+            ... on ParagraphBasicAccordion {
+              fieldHeader
+              fieldRichWysiwyg {
+                processed
+              }
+            }
+          }
+        }        
```

Comment:
We have some shared paragraph fragments here, `src/site/stages/build/drupal/graphql/fragments.graphql.js`. If you find yourself repeating this block, might be worth defining a fragment for `ParagraphBasicAccordion`

`src/site/stages/build/drupal/graphql/page.graphql.js` is an example of then importing the fragments into your layout query